pages/thanks-rider.html

```diff
@@ -8,17 +8,28 @@
     <div class="wrapper single-column offset-top">
         <div class="bannerbox">
             <div class="bannerbox__content">
-                <h1>Congratulations</h1>
-                <p>on taking this step to claim your vote! We’ve sent you a message confirming that we’ve received your request.</p>
-                <p>Your ride request reference number is: <b class="uuid">ID not loaded, please try again</b>.</p>
-                <p>Please <strong>keep this reference</strong> in case you need to manage your ride request.</p>
-                <p>To view or manage your ride request, visit our <b><a class="self-service-url" href="../self-service/?type=rider">self-service portal</a></b>.</p>
+                <h1 id="thanks-header">Congratulations</h1>
+                <span id="sorryHeading" hidden style="display:none">We are sorry</span>
+                
+                <p id="introPara">on taking this step to claim your vote! We’ve sent you a message confirming that we’ve received your request.</p>
+                <span id="sorryPara" hidden style="display:none">It has not been possible to process your request. Please see the information below for more details. We want to help you, so please contact our phone support if you cannot resolve this issue yourself.</span>
```

Comment:
It's a change suggestion to simplify the code. When I said "I've also changed the spans" etc., I meant in the change suggestion in my comment.

I was suggesting that you make these changes to your PR, but if you prefer I can make the changes to your PR myself. I would need you to give me write permissions on your fork, else I'll need to make a PR to your branch. 
Lastly, if you'd prefer to ship this ASAP without the suggested changes then I can make them in a future PR, but I'd rather do them all as part of a single PR.